website/views.py

```diff
@@ -191,23 +192,27 @@ def get_dashboard_nodes(auth):
     """
     user = auth.user
 
-    contributed = user.node__contributed  # nodes user contributed to
-
-    nodes = contributed.find(
-        Q('category', 'eq', 'project') &
-        Q('is_deleted', 'eq', False) &
-        Q('is_registration', 'eq', False) &
-        Q('is_folder', 'eq', False)
+    nodes = Node.find_for_user(
+        user,
+        subquery=(
+            Q('category', 'eq', 'project') &
```

Comment:
I know this is traditionally correct, and I'm not sure what's relying on this, but this is technically not true any more. Category can be project for subcomponents. I would recommend switching this to checking for a null parent instead. @sloria, so you agree?